docs/program/extend/SDK-as-server.md

```diff
@@ -1,31 +1,21 @@
 ---
-title: "Use SDKs to extend Viam"
-linkTitle: "Custom Resources with SDKs"
+title: "Add Custom Components as Remotes of Your Robot"
+linkTitle: "Custom Components as Remotes"
 weight: 99
 type: "docs"
 tags: ["server", "sdk"]
+description: "Implement custom components and register them on a server configured as a remote of your robot."
+webmSrc: "/tutorials/img/custom-base-dog/base-control-dog.webm"
+mp4Src: "/tutorials/img/custom-base-dog/base-control-dog.mp4"
+videoAlt: "A quadrupedal robot comprised of small servos, black laser cut acrylic, and with ultrasonic sensors for eyes, walks forward, backward, and turns from side to side on a desk. Next to it is a laptop with the robot's Control tab on the Viam app open in a browser window."
 ---
 
-Viam's Robot Development Kit ([RDK](/internals/rdk/)) provides a variety of components and services out of the box.
-If the components you are working with are not supported, you can use Viam [modular resources](/program/extend/) to add support for custom resources which form a seamless extension of the Viam platform.
+If a type or model of [component](/components) you are working with is not built into the [Viam RDK](/internals/rdk), you can use a [Viam SDK](/program/sdk-as-client) to code a custom resource implementation, host it on a server, and add it as a [remote](/manage/parts-and-remotes) of your robot.
```

Comment:
So, the text being replaced stated that the reason you'd do this is if you were not using go, and you'd therefore need to use this method instead of a modular resource.  The new text is not clear on why they'd choose this vs modular resource.  Can we make that clear?  I honestly am still not sure myself.

Reply:
@mcvella At the moment, they would choose this if they didn't know how to implement a modular resource or if they were following a tutorial with this method. In the future, I believe eng. wants this method to be completely deprecated in favor of modular resources, but that's pending full implementation of the registry?

Reply:
I could add a note saying that this method will be deprecated in favor of modular resources but that support is still ongoing? I'm not really sure how to handle this either. I'll see who I can bring in to review

Reply:
Is that what is happening? If so that makes sense as an explanation

Reply:
@npentrel @mcvella I added a note above (lines 13-16), but I am bringing in @Otterverse as well to confirm what future deprecation will look like

Reply:
Module support was initially only available in Go, so that's why there was the previous wording. Now that module support is in Python, it's preferred compared to setting up a remote.

SDK-as-server should be rare-if ever (once an SDK has module support) on most normal linux platforms. The cases where an SDK as a remote will still be preferred are basically Non-linux platforms (what the micro-sdk is going to support) or possibly other rare cases where the viam-server itself cannot run or be compiled.